projects/rero/ng-core/src/lib/record/detail/detail.component.ts

```diff
@@ -153,7 +154,7 @@ export class DetailComponent implements OnInit {
   public deleteRecord(pid: string) {
     this.recordUiService.deleteRecord(this.route.snapshot.paramMap.get('type'), pid).subscribe((result: any) => {
       if (result === true) {
-        this.location.back();
+        this.router.navigate(['../..'], {relativeTo: this.route});
```

Comment:
can you add a small comment as it is not easy to understand. For ex. back to the list or similar.